SingleBrokerDataDeletionTest.shouldNotCompactNotExportedEvents flaky

**Summary**

- How often does the test fail? at least once so far
- Does it block your work? no
- Do we suspect that it is a real failure? unknown

**Failures**

<details><summary>Example assertion failure</summary>
<pre>
<![CDATA[io.zeebe.util.exception.UncheckedExecutionException: Cluster start failed
	at io.zeebe.broker.it.clustering.ClusteringRule.before(ClusteringRule.java:224)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:50)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:55)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:137)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeLazy(JUnitCoreWrapper.java:119)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:87)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)
	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:157)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
Caused by: io.zeebe.client.api.command.ClientStatusException: No brokers available
	at io.zeebe.client.impl.ZeebeClientFutureImpl.transformExecutionException(ZeebeClientFutureImpl.java:93)
	at io.zeebe.client.impl.ZeebeClientFutureImpl.join(ZeebeClientFutureImpl.java:50)
	at io.zeebe.broker.it.clustering.ClusteringRule.lambda$getTopologyFromClient$7(ClusteringRule.java:387)
	at org.awaitility.core.AbstractHamcrestCondition.lambda$new$0(AbstractHamcrestCondition.java:48)
	at org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:222)
	at org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:209)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.util.concurrent.ExecutionException: io.grpc.StatusRuntimeException: UNAVAILABLE: No brokers available
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
	at io.zeebe.client.impl.ZeebeClientFutureImpl.join(ZeebeClientFutureImpl.java:48)
	... 8 more
Caused by: io.grpc.StatusRuntimeException: UNAVAILABLE: No brokers available
	at io.grpc.Status.asRuntimeException(Status.java:533)
	at io.grpc.stub.ClientCalls$StreamObserverToCallListenerAdapter.onClose(ClientCalls.java:460)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:426)
	at io.grpc.internal.ClientCallImpl.access$500(ClientCallImpl.java:66)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:689)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$900(ClientCallImpl.java:577)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:751)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:740)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	... 3 more
</pre>
</details>

**Hypotheses**

**Logs**


<details><summary>Logs</summary>
<pre>
20:44:49.642 [] [main] INFO  io.zeebe.test - Test started: shouldNotCompactNotExportedEvents(io.zeebe.broker.it.clustering.SingleBrokerDataDeletionTest)
20:44:49.644 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 28847 for test fork 3 with range PortRange{host='localhost', basePort=28800, maxOffset=100, currentOffset=148}
20:44:49.645 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 28848 for test fork 3 with range PortRange{host='localhost', basePort=28800, maxOffset=100, currentOffset=149}
20:44:49.645 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 28849 for test fork 3 with range PortRange{host='localhost', basePort=28800, maxOffset=100, currentOffset=150}
20:44:49.645 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 28850 for test fork 3 with range PortRange{host='localhost', basePort=28800, maxOffset=100, currentOffset=151}
20:44:49.645 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 28851 for test fork 3 with range PortRange{host='localhost', basePort=28800, maxOffset=100, currentOffset=152}
20:44:49.645 [] [main] DEBUG io.zeebe.broker.system - Initializing system with base path /tmp/junit8073152190480997519/0
20:44:49.647 [] [Thread-3506] INFO  io.zeebe.broker.system - Version: 0.25.0-SNAPSHOT
20:44:49.647 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 28852 for test fork 3 with range PortRange{host='localhost', basePort=28800, maxOffset=100, currentOffset=153}
20:44:49.647 [] [main] INFO  io.zeebe.test.util.SocketUtil - Choosing next port 28853 for test fork 3 with range PortRange{host='localhost', basePort=28800, maxOffset=100, currentOffset=154}
20:44:49.648 [] [Thread-3506] INFO  io.zeebe.broker.system - Starting broker 0 with configuration {
  "network" : {
    "host" : "0.0.0.0",
    "portOffset" : 0,
    "maxMessageSize" : "0MB",
    "advertisedHost" : "0.0.0.0",
    "commandApi" : {
      "host" : "0.0.0.0",
      "port" : 28849,
      "advertisedHost" : "0.0.0.0",
      "advertisedPort" : 28849,
      "address" : "0.0.0.0:28849",
      "advertisedAddress" : "0.0.0.0:28849"
    },
    "internalApi" : {
      "host" : "0.0.0.0",
      "port" : 28850,
      "advertisedHost" : "0.0.0.0",
      "advertisedPort" : 28850,
      "address" : "0.0.0.0:28850",
      "advertisedAddress" : "0.0.0.0:28850"
    },
    "monitoringApi" : {
      "host" : "0.0.0.0",
      "port" : 28851,
      "advertisedHost" : "0.0.0.0",
      "advertisedPort" : 28851,
      "address" : "0.0.0.0:28851",
      "advertisedAddress" : "0.0.0.0:28851"
    },
    "maxMessageSizeInBytes" : 8192
  },
  "cluster" : {
    "initialContactPoints" : [ "0.0.0.0:28850" ],
    "partitionIds" : [ 1 ],
    "nodeId" : 0,
    "partitionsCount" : 1,
    "replicationFactor" : 1,
    "clusterSize" : 1,
    "clusterName" : "zeebe-cluster-8",
    "membership" : {
      "broadcastUpdates" : false,
      "broadcastDisputes" : true,
      "notifySuspect" : false,
      "gossipInterval" : "PT0.15S",
      "gossipFanout" : 2,
      "probeInterval" : "PT0.25S",
      "probeTimeout" : "PT2S",
      "suspectProbes" : 3,
      "failureTimeout" : "PT2S",
      "syncInterval" : "PT10S"
    }
  },
  "threads" : {
    "cpuThreadCount" : 2,
    "ioThreadCount" : 2
  },
  "data" : {
    "directories" : [ "/tmp/junit8073152190480997519/0/data" ],
    "logSegmentSize" : "0MB",
    "snapshotPeriod" : "PT5M",
    "logIndexDensity" : 5,
    "logSegmentSizeInBytes" : 8192,
    "atomixStorageLevel" : "DISK"
  },
  "exporters" : {
    "snapshot-test-exporter" : {
      "jarPath" : null,
      "className" : "io.zeebe.broker.it.clustering.SingleBrokerDataDeletionTest$ControllableExporter",
      "args" : null,
      "external" : false
    }
  },
  "gateway" : {
    "network" : {
      "host" : "0.0.0.0",
      "port" : 28847,
      "minKeepAliveInterval" : "PT30S"
    },
    "cluster" : {
      "contactPoint" : "0.0.0.0:28850",
      "requestTimeout" : "PT15S",
      "clusterName" : "zeebe-cluster",
      "memberId" : "gateway",
      "host" : "0.0.0.0",
      "port" : 28848,
      "membership" : {
        "broadcastUpdates" : false,
        "broadcastDisputes" : true,
        "notifySuspect" : false,
        "gossipInterval" : "PT0.25S",
        "gossipFanout" : 2,
        "probeInterval" : "PT1S",
        "probeTimeout" : "PT2S",
        "suspectProbes" : 3,
        "failureTimeout" : "PT10S",
        "syncInterval" : "PT10S"
      }
    },
    "threads" : {
      "managementThreads" : 1
    },
    "monitoring" : {
      "enabled" : false,
      "host" : "0.0.0.0",
      "port" : 9600
    },
    "security" : {
      "enabled" : false,
      "certificateChainPath" : null,
      "privateKeyPath" : null
    },
    "longPolling" : {
      "enabled" : true
    },
    "initialized" : true,
    "enable" : false
  },
  "backpressure" : {
    "enabled" : true,
    "algorithm" : "VEGAS",
    "aimd" : {
      "requestTimeout" : "PT1S",
      "initialLimit" : 100,
      "minLimit" : 1,
      "maxLimit" : 1000,
      "backoffRatio" : 0.9
    },
    "fixedLimit" : {
      "limit" : 20
    },
    "vegas" : {
      "alpha" : 3,
      "beta" : 6,
      "initialLimit" : 20
    },
    "gradient" : {
      "minLimit" : 10,
      "initialLimit" : 20,
      "rttTolerance" : 2.0
    },
    "gradient2" : {
      "minLimit" : 10,
      "initialLimit" : 20,
      "rttTolerance" : 2.0,
      "longWindow" : 600
    }
  },
  "stepTimeout" : "PT5M",
  "executionMetricsExporterEnabled" : false
}
20:44:49.648 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [1/10]: actor scheduler
20:44:49.649 [] [Thread-3506] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [1/10]: actor scheduler started in 1 ms
20:44:49.649 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [2/10]: membership and replication protocol
20:44:49.649 [] [Thread-3506] DEBUG io.zeebe.broker.clustering - Member 0 will contact node: 0.0.0.0:28850
20:44:49.652 [] [Thread-3506] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [2/10]: membership and replication protocol started in 3 ms
20:44:49.652 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [3/10]: command api transport
20:44:49.653 [] [main] INFO  io.atomix.core.Atomix - 0.25.0-SNAPSHOT (revision 13badd built on 2020-07-07 20:36:22)
20:44:49.654 [] [netty-messaging-event-epoll-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:28849
20:44:49.655 [] [netty-messaging-event-epoll-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started
20:44:49.655 [] [Thread-3506] DEBUG io.zeebe.broker.system - Bound command API to 0.0.0.0:28849 
20:44:49.655 [] [Thread-3506] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [3/10]: command api transport started in 3 ms
20:44:49.655 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [4/10]: command api handler
20:44:49.655 [] [netty-messaging-event-epoll-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:28853
20:44:49.655 [] [Thread-3506] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [4/10]: command api handler started in 0 ms
20:44:49.655 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [5/10]: subscription api
20:44:49.655 [] [netty-messaging-event-epoll-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started
20:44:49.655 [] [Thread-3506] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [5/10]: subscription api started in 0 ms
20:44:49.655 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [6/10]: cluster services
20:44:49.655 [] [Thread-3506] INFO  io.atomix.core.Atomix - 0.25.0-SNAPSHOT (revision 13badd built on 2020-07-07 20:36:22)
20:44:49.657 [] [netty-unicast-event-nio-client-0] INFO  io.atomix.cluster.messaging.impl.NettyUnicastService - UDP server listening for connections on 0.0.0.0:28853
20:44:49.657 [] [atomix-cluster-0] INFO  io.atomix.cluster.discovery.BootstrapDiscoveryProvider - Joined
20:44:49.657 [] [netty-messaging-event-epoll-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - TCP server listening for connections on 0.0.0.0:28850
20:44:49.657 [] [netty-messaging-event-epoll-server-0] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Started
20:44:49.657 [] [atomix-cluster-0] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - Nodes from discovery service [Node{id=dff81726-13bf-4dc1-9979-eb205da4096b, address=0.0.0.0:28850}]
20:44:49.658 [] [atomix-cluster-0] INFO  io.atomix.cluster.protocol.SwimMembershipProtocol - Started
20:44:49.658 [] [atomix-cluster-0] INFO  io.atomix.cluster.impl.DefaultClusterMembershipService - Started
20:44:49.658 [] [atomix-cluster-0] INFO  io.atomix.cluster.messaging.impl.DefaultClusterCommunicationService - Started
20:44:49.658 [] [atomix-cluster-0] INFO  io.atomix.cluster.messaging.impl.DefaultClusterEventService - Started
20:44:49.658 [] [atomix-0] DEBUG io.atomix.primitive.partition.impl.DefaultPartitionService - Started class io.atomix.primitive.partition.impl.DefaultPartitionService
20:44:49.659 [] [netty-unicast-event-nio-client-0] INFO  io.atomix.cluster.messaging.impl.NettyUnicastService - UDP server listening for connections on 0.0.0.0:28850
20:44:49.659 [] [atomix-cluster-0] INFO  io.atomix.cluster.discovery.BootstrapDiscoveryProvider - Joined
20:44:49.659 [] [atomix-cluster-0] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - Nodes from discovery service [Node{id=8c6c704a-1436-4fe3-bdb4-12db60730422, address=0.0.0.0:28850}]
20:44:49.660 [] [atomix-cluster-0] INFO  io.atomix.cluster.protocol.SwimMembershipProtocol - Started
20:44:49.660 [] [atomix-cluster-0] INFO  io.atomix.cluster.impl.DefaultClusterMembershipService - Started
20:44:49.660 [] [atomix-cluster-0] INFO  io.atomix.cluster.messaging.impl.DefaultClusterCommunicationService - Started
20:44:49.660 [] [atomix-cluster-0] INFO  io.atomix.cluster.messaging.impl.DefaultClusterEventService - Started
20:44:49.660 [] [main] INFO  io.zeebe.gateway - Version: 0.25.0-SNAPSHOT
20:44:49.660 [] [atomix-0] INFO  io.atomix.raft.partition.impl.RaftPartitionServer - Starting server for partition PartitionId{id=1, group=raft-partition}
20:44:49.661 [] [main] INFO  io.zeebe.gateway - Starting gateway with configuration {
  "network" : {
    "host" : "0.0.0.0",
    "port" : 28852,
    "minKeepAliveInterval" : "PT30S"
  },
  "cluster" : {
    "contactPoint" : "0.0.0.0:28850",
    "requestTimeout" : "PT45S",
    "clusterName" : "zeebe-cluster-8",
    "memberId" : "gateway",
    "host" : "0.0.0.0",
    "port" : 28853,
    "membership" : {
      "broadcastUpdates" : false,
      "broadcastDisputes" : true,
      "notifySuspect" : false,
      "gossipInterval" : "PT0.25S",
      "gossipFanout" : 2,
      "probeInterval" : "PT1S",
      "probeTimeout" : "PT2S",
      "suspectProbes" : 3,
      "failureTimeout" : "PT10S",
      "syncInterval" : "PT10S"
    }
  },
  "threads" : {
    "managementThreads" : 1
  },
  "monitoring" : {
    "enabled" : false,
    "host" : "0.0.0.0",
    "port" : 9600
  },
  "security" : {
    "enabled" : false,
    "certificateChainPath" : null,
    "privateKeyPath" : null
  },
  "longPolling" : {
    "enabled" : true
  },
  "initialized" : true
}
20:44:49.673 [] [atomix-0] DEBUG io.atomix.storage.journal.SegmentedJournal - Created segment: JournalSegment{id=1, version=1, index=1}
20:44:49.677 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Transitioning to FOLLOWER
20:44:49.677 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.roles.FollowerRole - RaftServer{raft-partition-partition-1}{role=FOLLOWER} - Single member cluster. Transitioning directly to candidate.
20:44:49.677 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Transitioning to CANDIDATE
20:44:49.677 [] [raft-server-0-raft-partition-partition-1] WARN  io.atomix.utils.event.ListenerRegistry - Listener io.atomix.raft.roles.FollowerRole$$Lambda$317/0x00000008403b3c40@4ebd7618 not registered
20:44:49.678 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.roles.CandidateRole - RaftServer{raft-partition-partition-1}{role=CANDIDATE} - Single member cluster. Transitioning directly to leader.
20:44:49.683 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Set term 1
20:44:49.686 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Voted for 0
20:44:49.686 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Transitioning to LEADER
20:44:49.686 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Found leader 0
20:44:49.692 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.raft.impl.DefaultRaftServer - RaftServer{raft-partition-partition-1} - Server started successfully!
20:44:49.692 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.raft.partition.impl.RaftPartitionServer - Successfully started server for partition PartitionId{id=1, group=raft-partition}
20:44:49.692 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.partition.RaftPartitionGroup - Started
20:44:49.692 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.primitive.partition.impl.DefaultPartitionService - Started class io.atomix.primitive.partition.impl.DefaultPartitionService
20:44:49.692 [] [Thread-3506] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [6/10]: cluster services started in 37 ms
20:44:49.692 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [7/10]: topology manager
20:44:49.693 [] [Thread-3506] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [7/10]: topology manager started in 0 ms
20:44:49.693 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [8/10]: monitoring services
20:44:49.715 [] [Thread-3506] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [8/10]: monitoring services started in 22 ms
20:44:49.715 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [9/10]: leader management request handler
20:44:49.715 [] [Thread-3506] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [9/10]: leader management request handler started in 0 ms
20:44:49.715 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 [10/10]: zeebe partitions
20:44:49.716 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 partitions [1/1]: partition 1
20:44:49.716 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Removing follower partition service for partition PartitionId{id=1, group=raft-partition}
20:44:49.716 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Partition role transitioning from null to LEADER
20:44:49.716 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Installing leader partition service for partition PartitionId{id=1, group=raft-partition}
20:44:49.810 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - 0 - Detected local properties change {brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAAwAAA8AAAAwLjI1LjAtU05BUFNIT1Q=}
20:44:49.908 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Detected local properties change {event-service-topics-subscribed=HwEBAwFqb2JzQXZhaWxhYmzl}
20:44:50.419 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.logstreams.snapshot - Opened database from '/tmp/junit8073152190480997519/0/data/raft-partition/partitions/1/runtime'.
20:44:50.420 [Broker-0-LogStream-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.logstreams - Configured log appender back pressure at partition 1 as AppenderVegasCfg{initialLimit=1024, maxConcurrency=32768, alphaLimit=0.7, betaLimit=0.95}. Window limiting is disabled
20:44:50.420 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.logstreams - Recovering state of partition 1 from snapshot
20:44:50.420 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-1] INFO  org.camunda.feel.FeelEngine - Engine created. [value-mapper: CompositeValueMapper(List(io.zeebe.el.impl.feel.MessagePackValueMapper@3cd381f5)), function-provider: io.zeebe.el.impl.feel.FeelFunctionProvider@2df95c82, configuration: Configuration(false)]
20:44:50.421 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-1] INFO  io.zeebe.logstreams - Recovered state of partition 1 from snapshot at position -1
20:44:50.421 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-1] INFO  org.camunda.feel.FeelEngine - Engine created. [value-mapper: CompositeValueMapper(List(io.zeebe.el.impl.feel.MessagePackValueMapper@48971ccf)), function-provider: io.zeebe.el.impl.feel.FeelFunctionProvider@68ebf001, configuration: Configuration(false)]
20:44:50.422 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-1] INFO  org.camunda.feel.FeelEngine - Engine created. [value-mapper: CompositeValueMapper(List(io.zeebe.el.impl.feel.MessagePackValueMapper@f91acd2)), function-provider: io.zeebe.el.impl.feel.FeelFunctionProvider@7b932d51, configuration: Configuration(false)]
20:44:50.423 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-1] DEBUG io.zeebe.broker.exporter - Recovering exporter from snapshot
20:44:50.423 [Broker-0-HealthCheckService] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - All partitions are installed. Broker is ready!
20:44:50.424 [] [Thread-3506] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 partitions [1/1]: partition 1 started in 708 ms
20:44:50.424 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 partitions succeeded. Started 1 steps in 708 ms.
20:44:50.424 [] [Thread-3506] DEBUG io.zeebe.broker.system - Bootstrap Broker-0 [10/10]: zeebe partitions started in 708 ms
20:44:50.424 [] [Thread-3506] INFO  io.zeebe.broker.system - Bootstrap Broker-0 succeeded. Started 10 steps in 776 ms.
20:44:50.424 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-1] DEBUG io.zeebe.broker.exporter - Recovered exporter 'Broker-0-Exporter-1' from snapshot at lastExportedPosition -1
20:44:50.424 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-1] DEBUG io.zeebe.broker.exporter - Configure exporter with id 'snapshot-test-exporter'
20:44:50.424 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-1] DEBUG io.zeebe.broker.exporter - Set event filter for exporters: ExporterEventFilter{acceptRecordTypes={COMMAND_REJECTION=true, EVENT=true, COMMAND=true, SBE_UNKNOWN=true, NULL_VAL=true}, acceptValueTypes={NULL_VAL=true, SBE_UNKNOWN=true, MESSAGE=true, TIMER=true, DEPLOYMENT=true, WORKFLOW_INSTANCE=true, WORKFLOW_INSTANCE_RESULT=true, WORKFLOW_INSTANCE_SUBSCRIPTION=true, MESSAGE_SUBSCRIPTION=true, ERROR=true, INCIDENT=true, WORKFLOW_INSTANCE_CREATION=true, JOB_BATCH=true, VARIABLE_DOCUMENT=true, VARIABLE=true, JOB=true, MESSAGE_START_EVENT_SUBSCRIPTION=true}}
20:44:50.424 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-1] DEBUG io.zeebe.broker.exporter - Open exporter with id 'snapshot-test-exporter'
20:44:50.566 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - 0 - Detected local properties change {brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}
20:44:50.634 [] [main] DEBUG io.zeebe.broker.system - Closing ClusteringRule...
20:44:50.634 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [1/10]: zeebe partitions
20:44:50.634 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 partitions [1/1]: partition 1
20:44:50.634 [Broker-0-Exporter-1] [Broker-0-zb-fs-workers-1] DEBUG io.zeebe.broker.exporter - Closed exporter director 'Broker-0-Exporter-1'.
20:44:50.634 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.exporter.stream.ExporterDirector@3ebb7974 successfully
20:44:50.635 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$583/0x0000000840521840@5c202257 successfully
20:44:50.635 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.impl.AsyncSnapshotDirector@4694e03c successfully
20:44:50.635 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$370/0x00000008403e2c40@fbbb350 successfully
20:44:50.635 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.logstreams - Closed stream processor controller Broker-0-StreamProcessor-1.
20:44:50.635 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.engine.processor.StreamProcessor@6a2e0b0a successfully
20:44:50.635 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$382/0x00000008403e4040@3741ae82 successfully
20:44:50.635 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.logstreams.LogDeletionService@56353d56 successfully
20:44:50.661 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - 0 - Member added Member{id=gateway, address=0.0.0.0:28853, properties={event-service-topics-subscribed=HwEBAwFqb2JzQXZhaWxhYmzl}}
20:44:50.662 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Member added Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}}
20:44:50.662 [GatewayTopologyManager] [gateway-scheduler-zb-actors-0] DEBUG io.zeebe.gateway - Received new broker BrokerInfo{nodeId=0, partitionsCount=1, clusterSize=1, replicationFactor=1, partitionRoles={1=LEADER}, partitionLeaderTerms={1=1}, version=0.25.0-SNAPSHOT}.
20:44:50.680 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.logstreams.snapshot - Closed database from '/tmp/junit8073152190480997519/0/data/raft-partition/partitions/1/runtime'.
20:44:50.680 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$380/0x00000008403e4840@44635f3 successfully
20:44:50.680 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$372/0x00000008403e3440@1e2a082a successfully
20:44:50.680 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$371/0x00000008403e3040@242590d3 successfully
20:44:50.680 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$370/0x00000008403e2c40@5b85d68c successfully
20:44:50.680 [Broker-0-LogStream-1] [Broker-0-zb-actors-0] INFO  io.zeebe.logstreams - Close appender for log stream logstream-raft-partition-partition-1
20:44:50.680 [logstream-raft-partition-partition-1-write-buffer] [Broker-0-zb-actors-0] DEBUG io.zeebe.dispatcher - Dispatcher closed
20:44:50.680 [Broker-0-LogStream-1] [Broker-0-zb-actors-0] INFO  io.zeebe.logstreams - On closing logstream logstream-raft-partition-partition-1 close 3 readers
20:44:50.680 [Broker-0-LogStream-1] [Broker-0-zb-actors-0] INFO  io.zeebe.logstreams - Close log storage with name logstream-raft-partition-partition-1
20:44:50.681 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-0] DEBUG io.zeebe.broker.system - Closed io.zeebe.broker.system.partitions.ZeebePartition$$Lambda$368/0x00000008403e2440@23a53d17 successfully
20:44:50.681 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 partitions [1/1]: partition 1 closed in 47 ms
20:44:50.681 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 partitions succeeded. Closed 1 steps in 47 ms.
20:44:50.681 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [1/10]: zeebe partitions closed in 47 ms
20:44:50.681 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [2/10]: leader management request handler
20:44:50.681 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [2/10]: leader management request handler closed in 0 ms
20:44:50.681 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [3/10]: monitoring services
20:44:50.681 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [3/10]: monitoring services closed in 0 ms
20:44:50.681 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [4/10]: topology manager
20:44:50.681 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [4/10]: topology manager closed in 0 ms
20:44:50.681 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [5/10]: cluster services
20:44:50.681 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [5/10]: cluster services closed in 0 ms
20:44:50.681 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [6/10]: subscription api
20:44:50.681 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [6/10]: subscription api closed in 0 ms
20:44:50.681 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [7/10]: command api handler
20:44:50.681 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [7/10]: command api handler closed in 0 ms
20:44:50.681 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [8/10]: command api transport
20:44:51.715 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Start synchronizing membership with Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}, version=0.25.0-SNAPSHOT, timestamp=1594154689659, state=ALIVE, incarnationNumber=1594154689661}
20:44:51.721 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Finished synchronizing membership with Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}, version=0.25.0-SNAPSHOT, timestamp=1594154689659, state=ALIVE, incarnationNumber=1594154689661}, received: '[Member{id=gateway, address=0.0.0.0:28853, properties={event-service-topics-subscribed=HwEBAwFqb2JzQXZhaWxhYmzl}, version=0.25.0-SNAPSHOT, timestamp=1594154689657, state=ALIVE, incarnationNumber=1594154689658}, Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}, version=0.25.0-SNAPSHOT, timestamp=1594154689659, state=ALIVE, incarnationNumber=1594154689661}]'
20:44:52.721 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Start synchronizing membership with Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}, version=0.25.0-SNAPSHOT, timestamp=1594154689659, state=ALIVE, incarnationNumber=1594154689661}
20:44:52.722 [] [ForkJoinPool.commonPool-worker-5] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Stopped
20:44:52.723 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [8/10]: command api transport closed in 2042 ms
20:44:52.723 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [9/10]: membership and replication protocol
20:44:52.725 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.impl.RaftContext - RaftServer{raft-partition-partition-1} - Transitioning to INACTIVE
20:44:52.732 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.raft.impl.zeebe.ZeebeRaftStateMachine - ZeebeRaftStateMachine{raft-partition-partition-1} - Closing state machine raft-partition-partition-1
20:44:52.732 [] [raft-server-0-raft-partition-partition-1] DEBUG io.atomix.storage.journal.SegmentedJournal - Closing segment: JournalSegment{id=1, version=1, index=1}
20:44:52.733 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.raft.partition.RaftPartitionGroup - Stopped
20:44:52.733 [] [raft-server-0-raft-partition-partition-1] INFO  io.atomix.primitive.partition.impl.DefaultPartitionService - Stopped
20:44:52.733 [] [atomix-0] INFO  io.atomix.cluster.messaging.impl.DefaultClusterCommunicationService - Stopped
20:44:52.733 [] [atomix-cluster-0] INFO  io.atomix.cluster.messaging.impl.DefaultClusterEventService - Stopped
20:44:52.733 [] [atomix-cluster-0] INFO  io.atomix.cluster.protocol.SwimMembershipProtocol - 0 - Member deactivated: Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}}
20:44:52.733 [] [atomix-cluster-0] INFO  io.atomix.cluster.protocol.SwimMembershipProtocol - Stopped
20:44:52.733 [] [atomix-cluster-0] INFO  io.atomix.cluster.discovery.BootstrapDiscoveryProvider - Left
20:44:52.733 [] [atomix-cluster-0] INFO  io.atomix.cluster.impl.DefaultClusterMembershipService - Stopped
20:44:52.735 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Failed to synchronize membership with Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}, version=0.25.0-SNAPSHOT, timestamp=1594154689659, state=ALIVE, incarnationNumber=1594154689661}
io.atomix.cluster.messaging.MessagingException$NoRemoteHandler: No remote message handler registered for this message
	at io.atomix.cluster.messaging.impl.AbstractClientConnection.dispatch(AbstractClientConnection.java:64) ~[atomix-cluster-0.25.0-SNAPSHOT.jar:0.25.0-SNAPSHOT]
	at io.atomix.cluster.messaging.impl.AbstractClientConnection.dispatch(AbstractClientConnection.java:37) ~[atomix-cluster-0.25.0-SNAPSHOT.jar:0.25.0-SNAPSHOT]
	at io.atomix.cluster.messaging.impl.NettyMessagingService$MessageDispatcher.channelRead0(NettyMessagingService.java:972) ~[atomix-cluster-0.25.0-SNAPSHOT.jar:0.25.0-SNAPSHOT]
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324) ~[netty-codec-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296) ~[netty-codec-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:792) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:475) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) ~[netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.50.Final.jar:4.1.50.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
20:44:52.748 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Failed to probe Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}, version=0.25.0-SNAPSHOT, timestamp=1594154689659, state=ALIVE, incarnationNumber=1594154689661}
io.atomix.cluster.messaging.MessagingException$NoRemoteHandler: No remote message handler registered for this message
	at io.atomix.cluster.messaging.impl.AbstractClientConnection.dispatch(AbstractClientConnection.java:64) ~[atomix-cluster-0.25.0-SNAPSHOT.jar:0.25.0-SNAPSHOT]
	at io.atomix.cluster.messaging.impl.AbstractClientConnection.dispatch(AbstractClientConnection.java:37) ~[atomix-cluster-0.25.0-SNAPSHOT.jar:0.25.0-SNAPSHOT]
	at io.atomix.cluster.messaging.impl.NettyMessagingService$MessageDispatcher.channelRead0(NettyMessagingService.java:972) ~[atomix-cluster-0.25.0-SNAPSHOT.jar:0.25.0-SNAPSHOT]
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324) ~[netty-codec-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296) ~[netty-codec-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:792) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:475) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) ~[netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.50.Final.jar:4.1.50.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
20:44:52.817 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Failed all probes of Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}}
20:44:52.817 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Member unreachable Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}}
20:44:52.817 [GatewayTopologyManager] [gateway-scheduler-zb-actors-0] DEBUG io.zeebe.gateway - Received REACHABILITY_CHANGED for broker 0, do nothing.
20:44:53.749 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Start synchronizing membership with Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}, version=0.25.0-SNAPSHOT, timestamp=1594154689659, state=SUSPECT, incarnationNumber=1594154689661}
20:44:53.816 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Failed to synchronize membership with Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}, version=0.25.0-SNAPSHOT, timestamp=1594154689659, state=SUSPECT, incarnationNumber=1594154689661}
io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /0.0.0.0:28850
Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
	at io.netty.channel.unix.Errors.throwConnectException(Errors.java:124) ~[netty-transport-native-unix-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.unix.Socket.finishConnect(Socket.java:243) ~[netty-transport-native-unix-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:672) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:649) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:529) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:465) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) ~[netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.50.Final.jar:4.1.50.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
20:44:53.818 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Failed to probe Member{id=dff81726-13bf-4dc1-9979-eb205da4096b, address=0.0.0.0:28850, properties={}, version=null, timestamp=0, state=null, incarnationNumber=0}
io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /0.0.0.0:28850
Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
	at io.netty.channel.unix.Errors.throwConnectException(Errors.java:124) ~[netty-transport-native-unix-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.unix.Socket.finishConnect(Socket.java:243) ~[netty-transport-native-unix-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:672) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:649) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:529) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:465) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) ~[netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.50.Final.jar:4.1.50.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
20:44:54.817 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Start synchronizing membership with Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}, version=0.25.0-SNAPSHOT, timestamp=1594154689659, state=SUSPECT, incarnationNumber=1594154689661}
20:44:54.819 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Failed to synchronize membership with Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}, version=0.25.0-SNAPSHOT, timestamp=1594154689659, state=SUSPECT, incarnationNumber=1594154689661}
io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /0.0.0.0:28850
Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
	at io.netty.channel.unix.Errors.throwConnectException(Errors.java:124) ~[netty-transport-native-unix-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.unix.Socket.finishConnect(Socket.java:243) ~[netty-transport-native-unix-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:672) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:649) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:529) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:465) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) ~[netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.50.Final.jar:4.1.50.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
20:44:54.822 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Failed to probe Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}, version=0.25.0-SNAPSHOT, timestamp=1594154689659, state=SUSPECT, incarnationNumber=1594154689661}
io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /0.0.0.0:28850
Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
	at io.netty.channel.unix.Errors.throwConnectException(Errors.java:124) ~[netty-transport-native-unix-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.unix.Socket.finishConnect(Socket.java:243) ~[netty-transport-native-unix-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:672) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:649) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:529) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:465) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378) ~[netty-transport-native-epoll-4.1.50.Final-linux-x86_64.jar:4.1.50.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) ~[netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.50.Final.jar:4.1.50.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
20:44:54.822 [] [atomix-cluster-heartbeat-sender] DEBUG io.atomix.cluster.protocol.SwimMembershipProtocol - gateway - Failed all probes of Member{id=0, address=0.0.0.0:28850, properties={brokerInfo=EADJAAAAAQAAAAAAAQAAAAEAAAABAAAAAAABCgAAAGNvbW1hbmRBcGkNAAAAMC4wLjAuMDoyODg0OQUAAQEAAAAADAABAQAAAAEAAAAAAAAADwAAADAuMjUuMC1TTkFQU0hPVA==}}
20:44:54.849 [] [ForkJoinPool.commonPool-worker-5] INFO  io.atomix.cluster.messaging.impl.NettyMessagingService - Stopped
20:44:54.849 [] [atomix-cluster-0] INFO  io.atomix.cluster.AtomixCluster - Stopped
20:44:54.849 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [9/10]: membership and replication protocol closed in 2126 ms
20:44:54.849 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 [10/10]: actor scheduler
20:44:54.854 [] [main] DEBUG io.zeebe.broker.system - Closing Broker-0 [10/10]: actor scheduler closed in 5 ms
20:44:54.854 [] [main] INFO  io.zeebe.broker.system - Closing Broker-0 succeeded. Closed 10 steps in 4220 ms.
20:44:54.854 [] [main] INFO  io.zeebe.broker.system - Broker shut down.
</pre>
</details>

--- a/qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/ClusteringRule.java
+++ b/qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/ClusteringRule.java
@@ -384,6 +384,7 @@ public final class ClusteringRule extends ExternalResource {
     return Awaitility.await()
         .pollInterval(Duration.ofMillis(100))
         .atMost(Duration.ofSeconds(10))
+        .ignoreExceptions()
         .until(() -> client.newTopologyRequest().send().join(), Objects::nonNull);
   }
 
@@ -600,6 +601,7 @@ public final class ClusteringRule extends ExternalResource {
     Awaitility.await()
         .pollInterval(Duration.ofMillis(100))
         .atMost(Duration.ofSeconds(10))
+        .ignoreExceptions()
         .until(() -> getTopologyFromClient().getBrokers(), topologyPredicate);
   }
 

